app/services/tax_host.rb

```diff
@@ -130,7 +130,7 @@ def check_for_orphans
       taxable_type = hash_key_to_class(key)
       unless array_values.empty?
         found_orphan = true
-        taxonomy.errors.add(taxable_type.tableize, _("you cannot remove %s that are used by hosts or inherited.") % taxable_type.tableize.humanize.downcase)
+        taxonomy.errors.add(taxable_type.tableize, _("expecting %s used by hosts or inherited (check mismatches report).") % _(taxable_type.tableize.humanize.downcase))
```

Comment:
`Check %s mismatches report` maybe? I'm trying to point the user to the mismatches report directly (if possible a link would be the best).